### ISSUE TITLE
hope to use it in conjunction with tailwindcss, similar to twin.macro

### ISSUE THREAD

**Issue by xh4528** [vue-styled-components/core#17]
### Is your feature request related to a problem? Please describe

Just like this, combining tailwind and styled-components

[twin.macro](https://github.com/ben-rogerson/twin.macro)

```
const StyledDiv = styled.div`
  width: 100px;
  height: 100px;
  background-color: #ccc;

  ${tw`text-black flex relative`}
`;
````

**Comment by akinoccc** [comment]
I need some time to investigate and I'm not sure if it can be done, but I will try my best.🥸

**Comment by akinoccc** [comment]
@xh4528 Now Tailwind CSS has been available since version 1.6.0. The usage see https://vue-styled-components.com/guide/advances/tailwind-css
> **Notes: The lib has renamed to `@vue-styled-components/core`**

**Comment by akinoccc** [comment]
> @xh4528 Now Tailwind CSS has been available since version 1.6.0. The usage see https://vue-styled-components.com/guide/advances/tailwind-css
> 
> > **Notes: The lib has renamed to `@vue-styled-components/core`**

If has any bug, welcome give feedback

**Comment by xh4528** [comment]
> > @xh4528 Now Tailwind CSS has been available since version 1.6.0. The usage see https://vue-styled-components.com/guide/advances/tailwind-css
> > > **Notes: The lib has renamed to `@vue-styled-components/core`**
> 
> If has any bug, welcome give feedback

Thank you, it's really great, the feature I've been looking forward to
